cluster/dragon/proc_mgr.go

```diff
@@ -220,6 +225,13 @@ func (p *procManager) scheduleBroadcast() {
 	p.broadcastTimer = time.AfterFunc(1*time.Second, p.broadcastInfo)
 }
 
+func (p *procManager) scheduleClusterAllocationsLogging() {
+	if !p.started {
+		return
+	}
+	p.custerLoggingTimer = time.AfterFunc(time.Minute, p.logClusterAllocations)
```

Comment:
This timer should be stopped in the Stop() method of ProcManager similar to how we do it with the broadcastTimer, otherwise will leak

Reply:
Good catch. I'll update.